build.yaml

```diff
@@ -347,6 +349,7 @@ common_cli:
     # bootstrapping. These are copied from the Jenkins Master /etc/slave_config
     # directory.
     export FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE=/tmp/acceptance.yaml
+    export FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_SECTION=storage-drivers.gce
     export FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER=gce
```

Comment:
Never-mind, I see this is used to get the minimum blockdevice size. I suppose we can also use this to create a thing that spins up a node and runs the tests at some point in the future.